### PR TITLE
PowerFactory DGS export configuration doc

### PR DESCRIPTION
**Pull request by geofjamg**
Signed-off-by: Geoffroy Jamgotchian <geoffroy.jamgotchian@gmail.com>

**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*
No

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Doc

**What is the new behavior (if this is a feature change)?**
A documentation has been to added to describe DGS export config required by PowSyBl importer.

**Does this PR introduce a breaking change or deprecate an API?** *If yes, check the following:*
- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

**Review by zamarrenolm — APPROVED** [review]
I've just made some suggestions for the grammar and comments about phase shifting in Common Impedances.

**Review by marqueslanauja — APPROVED** [review]

File: powerfactory/powerfactory-dgs/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+DGS reader
+----------
+
+PowSyBl only support PowerFactory DGS V5 and only the ASCII format. A basic DGS export configuration is provided with the
```
[the remark this answers is not in the document]
Comment (geofjamg) [reply powerfactory/powerfactory-dgs/README.md:4]: Done

File: powerfactory/powerfactory-dgs/README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+DGS reader
+----------
+
+PowSyBl only support PowerFactory DGS V5 and only the ASCII format. A basic DGS export configuration is provided with the 
+PowerFactory application in <PowerFactory install dir>\Dgs\V5.X\DGS Export Definitions 5.pfd. This file has to be imported 
+in your database. This basic configuration does not provide all necessary objects and attributes to PowSyBl converter to 
+achieve a correct conversion of the initial data model (and get a calculated state when running a load flow close to 
+one calculated by PowerFactory).
```
[the remark this answers is not in the document]
Comment (geofjamg) [reply powerfactory/powerfactory-dgs/README.md:8]: Done

File: powerfactory/powerfactory-dgs/README.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+
+
+Two and three windings transformer taps definition attributes need to be added to existing configuration. Optionally voltage magnitude and angle could be added to terminal to compare calculated state with PowerFactory one. VSC and common impedance configuration are totally missing and need to be created.
```
[the remark this answers is not in the document]
Comment (geofjamg) [reply powerfactory/powerfactory-dgs/README.md:12]: Done

File: powerfactory/powerfactory-dgs/README.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  - e:usetp
+  - e:Pnold
+  - e:Unom
+  - e:P_max
+
+- Common Impedance.IntMon (class ElmZpu)
+  - e:loc_name
+  - e:outserv
+  - e:Sn
+  - e:nphases
```
[the remark this answers is not in the document]
Comment (geofjamg) [reply powerfactory/powerfactory-dgs/README.md:39]: nphases removed.
So we keep nphshift and ag?

File: powerfactory/powerfactory-dgs/README.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+Two and three windings transformer taps definition attributes need to be added to existing configuration. Optionally voltage magnitude and angle could be added to terminal to compare calculated state with PowerFactory one. VSC and common impedance configuration are totally missing and need to be created.
+
+- 2-Winding Transformer.IntMon (class ElmTr2)
+   - e:mTaps
+
+- 3-Winding Transformer.IntMon (class ElmTr3)
+  - e:mTaps
+  - e:iMeasTap
+
```
[the remark this answers is not in the document]
Comment (geofjamg) [reply powerfactory/powerfactory-dgs/README.md:20]: I confirm, all theses attributes are already present in default config, at least the one provided by PF 2022.

File: powerfactory/powerfactory-dgs/README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (zamarrenolm) [reply powerfactory/powerfactory-dgs/README.md]: I think we can keep `nphshift` and `ag`. At some point it could be supported.

**Pull request merged**